src/python/pants/bin/BUILD

```diff
@@ -9,7 +9,6 @@ python_sources(
     dependencies=[":plugins"],
 )
 
-# NOTE: When adding a new backend, add it to `generate_docs.py` too! (If stable enough for docs)
```

Comment:
Reminder that the new workflow just enables everything.

One less list to ~rememeber~ forget to add to!